lib/tjbot.js

```diff
@@ -841,9 +858,21 @@ TJBot.prototype.speak = function(message) {
         throw new Error("TJBot tried to speak a null message.");
     }
 
+    //get voice
+    var selectedVoice = "en-US_MichaelVoice"; //default voice if is not possible to find a better one
```

Comment:
yeah... it is just an "internal default voice" if user set config.js with a wrong value. Example:
exports.config.voice = {"gender": "male", "language": "dn-US"};

dn-US does not exists... so it will use en-US as a default voice